proc-macros/src/lib.rs

```diff
@@ -310,3 +299,24 @@ fn rpc_param_name(pat: &syn::Pat, _attrs: &[syn::Attribute]) -> syn::parse::Resu
 		_ => unimplemented!(),
 	}
 }
+
+/// Search for `jsonrpsee` in `Cargo.toml`.
+fn find_jsonrpsee_crate() -> Result<proc_macro2::TokenStream, syn::Error> {
+	match crate_name("jsonrpsee") {
+		Ok(FoundCrate::Name(name)) => {
+			let ident = syn::Ident::new(&name, Span::call_site());
+			Ok(quote!(#ident::types))
+		}
+		Ok(FoundCrate::Itself) => panic!("Deriving RPC methods in any of the `jsonrpsee crates` is not supported"),
+		Err(_) => match (crate_name("jsonrpsee-http-client"), crate_name("jsonrpsee-ws-client")) {
```

Comment:
This was a bug before we searched only for the clients crates in `Cargo.toml`.

So now:
1. Search for `jsonrpsee crate`
2. if not found search for `client crates`